### PR TITLE
Explicit mention @ character as verbatim identifier

### PR DESCRIPTION
**Pull request by nachoalonsoportillo**

### CODE REVIEW

**Comment by Court72** [comment]
@shsagir

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: data-explorer/kusto/query/scalar-data-types/string.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ newline characters (`\n`), and itself (`\\`).
 
 ## Verbatim string literals
 
-Verbatim string literals are also supported. In this form, the backslash character (`\`) stands for itself, and not as an escape character.
+Verbatim string literals are also supported. In this form, the backslash character (`\`) stands for itself, and not as an escape character. Prepending the @ special character to a string literals, serves as a verbatim identifier.
```
Comment (shsagir) [inline data-explorer/kusto/query/scalar-data-types/string.md:36]:
```suggestion
Verbatim string literals are also supported. In this form, the backslash character (`\`) stands for itself, and not as an escape character. Prepending the `@` special character to string literals serves as a verbatim identifier.
```

Comment (shsagir) [inline data-explorer/kusto/query/scalar-data-types/string.md:36]:
```suggestion
Verbatim string literals are also supported. In this form, the backslash character (`\`) stands for itself, and not as an escape character. Prepending the `@` special character to string literals serves as a verbatim identifier
```

Comment (nachoalonsoportillo) [reply data-explorer/kusto/query/scalar-data-types/string.md:36]: Looks good to me.

**Comment by YaelSchuster** [comment]
Recreated in private repo to push

#please-close

**Pull request closed without merging**